src/models/indexable.php

```diff
@@ -26,7 +26,6 @@
  * @property string  $permalink
  * @property string  $permalink_hash
  * @property string  $canonical
- * @property int     $content_score
```

Comment:
Why did you remove this?

Reply:
This is a reference to an unexisting indexable field. The field has been renamed in the meanwhile and we don't do anything with it.